terminal/static/terminal.js

```diff
@@ -7,10 +7,11 @@ Terminal: false
 
 const terminal = new Terminal({ cursorBlink: true });
 const fitAddon = new FitAddon.FitAddon();
-const socket = io.connect("/terminal");
+const terminalPortString = window.location.pathname.split("/")[1]
+const socket = io("/terminal", {path: `/${terminalPortString}/socket.io`});
```

Comment:
doesn't work for me. what is the value of `terminalPortString` for you ? Is it the uuid4 generated by the back-end ?

Reply:
The terminalPortString value is "terminalXXXX" where XXXX is the port number of the flask server coordinating the SSH session.